compose/monitoring.yml

```diff
@@ -44,6 +45,21 @@ services:
       db:
         condition: service_started
     restart: unless-stopped
+  sql-exporter:
+    image: burningalchemist/sql_exporter
+    container_name: sql-exporter
+    expose:
+      - 9399
+    command: --config.file=/etc/sql-exporter/sql-exporter.yml
+    volumes:
+      - ./default/sql-exporter.yml:/etc/sql-exporter/sql-exporter.yml
+    depends_on:
+      prometheus:
+        condition: service_started
+      db:
+        condition: service_started
```

Comment:
That makes `sql-exporter` starts after these services, which is great.
Is `sql-exporter` robust enough to not stop if the database `musicbrainz` is not available yet?
Otherwise a custom condition or health check might be needed.

Reply:
I tested this (as per @mwiencek's suggestion) by taking the services down, renaming the database, taking them back up, and then after a while renaming the database back to `musicbrainz_db`. Both errored while unable to connect, but eventually connected back and started showing data again, so they seem resilient enough.